Dockerfile.fastbuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 The Kubernetes Authors.
```

Comment:
Not for this PR, @fabriziopandini came up with a good solution upstream to speed up builds using experimental/buildkit features, might be worth taking a look later.

Reply:
SGTM!